bisection.py

```diff
@@ -47,15 +47,15 @@ def targets_to_bmfilter(targets: List[str], models: List[str]) -> str:
         return "(not slomo)"
     for test in targets:
         regex = re.compile("test_(train|eval)\[([a-zA-Z0-9_]+)-([a-z]+)-([a-z]+)\]")
-        m = regex.match(test).groups()
+        m = regex.match(test)
         if not m:
             if m in models:
```

Comment:
hmm, is line 52 supposed to be 'if test in models'?

Reply:
Ah, right! Thanks for pointing out!